### PR TITLE
[NOT AZURE] Updated Azure Stack Infrastructure Insights Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

Recreated the FRP specifications and regenerated the SDK.  

https://github.com/Azure/azure-rest-api-specs/pull/2812

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/InfrastructureInsightsAdmin/InfrastructureInsights.Admin/Microsoft.AzureStack.Management.InfrastructureInsights.Admin.csproj [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
-    <VersionPrefix>0.1.0-preview</VersionPrefix>
-    <AssemblyName>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</AssemblyName>
-    <PackageTags>Microsoft Azure Stack;InfrastructureInsights;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
-    <PackageReleaseNotes>This is a preview release of the AzureStack InfrastructureInsights Admin SDK. Included with this release is support for gathering information about the health of your AzureStack infrastructure.</PackageReleaseNotes>
-  </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-  </PropertyGroup>
+    <PropertyGroup>
+        <PackageId>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</PackageId>
```
Comment (dsgouda) [inline src/AzureStack/InfrastructureInsightsAdmin/InfrastructureInsights.Admin/Microsoft.AzureStack.Management.InfrastructureInsights.Admin.csproj]: Looks like this package is already published and there are some additive changes being introduced. Please update the package version number

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good, just needs a version bump

**Comment by deathly809** [comment]
@dsgouda Done!  Also, do we publish the Nugets?  We might have to update the package release notes.

**Comment by dsgouda** [comment]
@deathly809 I believe so, please check [https://www.nuget.org/packages/Microsoft.AzureStack.Management.InfrastructureInsights.Admin/0.1.0-preview]

**Comment by deathly809** [comment]
@dsgouda I already updated the version.  So I have to kick off the build process correct?  Just to be 100% clear.

**Comment by dsgouda** [comment]
By build if you mean publish the package, then yes. The "build" itself was done successfully here by the CI machines.
What you may have to consider is updating the PackageReleaseNotes, let me know if you plan to do so

**Comment by deathly809** [comment]
@dsgouda I have been trying to get feedback on that.  We might just have a single PR to update all the admin PackageReleaseNotes and then kick of a publish job.

**Comment by dsgouda** [comment]
That would be a good idea. Will merge this for now, feel free to open a new PR for the release notes.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**